mex/editor/search/transform.py

```diff
@@ -12,8 +12,8 @@ def transform_models_to_results(
     return [
         SearchResult(
             identifier=model.identifier,
-            title=transform_models_to_title([model]),
-            preview=transform_models_to_preview([model]),
+            title=transform_models_to_title([model]),  # type: ignore[list-item]
+            preview=transform_models_to_preview([model]),  # type: ignore[list-item]
```

Comment:
```suggestion
            title=transform_models_to_title([model]),
            preview=transform_models_to_preview([model]),
```

shouldn't be necessary when those two functions allow `AnyPreviewModel`, see below..

Reply:
fixed in [a12e207](https://github.com/robert-koch-institut/mex-editor/pull/225/commits/a12e2071841ca1a0ec364aec538d260daf89e557)